docker/apps/agreement_database/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM vickumar/python-nodejs
+
+RUN git clone https://github.cfpb.gov/CFPB/agreement_database.git /apps/agreement_database/
```

Comment:
I don't think these URLs can live in the public GitHub. We've been thinking about how to handle requirement-specification for stuff that lives privately.